benchmarks/kernels/benchmark_paged_attention.py

```diff
@@ -9,8 +9,8 @@
 from vllm.utils import (STR_DTYPE_TO_TORCH_DTYPE, FlexibleArgumentParser,
                         create_kv_caches_with_random)
 
-NUM_BLOCKS = 1024 * 1024
-PARTITION_SIZE = 512
+NUM_BLOCKS = 256 * 1024
+PARTITION_SIZE = 256
```

Comment:
Can you explain what is the reason changing the values of the two constants? and is this change ROCm specific?

Reply:
We are also unsure about these changes in the parameters set by the original author. Is it possible to get in touch with the author to understand about his optimization and algorithm?